### PR TITLE
AccountRoutingTableDispatcher should reply with a SESSION_CLOSE message for unroutable USSD requests

### PR DESCRIPTION
**Pull request by hodgestar**
Currently if we get an unroutable USSD request we simply timeout, if we see the message is a USSD message then we should send a default closing reply so the connection to the phone is terminated instead of timing out.

### CODE REVIEW

File: go/vumitools/routing.py [hunk 1]
```diff
@@ -160,6 +160,10 @@ class AccountRoutingTableDispatcherConfig(RoutingTableDispatcher.CONFIG_CLASS,
         static=True, required=False)
     user_account_key = ConfigText(
         "Key of the user account the message is from.")
+    unroutable_inbound_session_reply = ConfigText(
+        "Text to send on encounter unroutable inbound messages"
+        " that are part of a session.",
```
Comment (jerith) [inline go/vumitools/routing.py]: Clumsy wording here. Perhaps "Text to send in response to unroutable inbound messages that are part of a session." instead?

**Comment by jerith** [comment]
Travis is unhappy with this and it looks like the failures are related to the changes in this branch.

**Comment by jerith** [comment]
I'm not hugely enthusiastic about a single static response to all the various flavours of sessions we may encounter. Perhaps the copy belongs in tagpool metadata with a fallback configured in the dispatcher?

**Comment by hodgestar** [comment]
Travis is unhappy because the Vumi side only just landed in Vumi develop, I
think?

**Comment by hodgestar** [comment]
I like the suggestion of being able to configure the message
per-tagpool. Will add.

**Comment by jerith** [comment]
Maybe tagpool metadata should also be allowed to override whether the replies are sent at all?

**Comment by hodgestar** [comment]
I'm struggling to think of a great use case for that off the top of my head
but I can see how it's useful to have theoretically. Bit wary of adding too
many knobs to tweak at the start. Would prefer to leave adding it as a
feature later, but happy to add it now if you feel it's really important.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**